README.md

```diff
@@ -39,47 +39,43 @@ The curator ensures the consistency of the id system by:
 - setting caps for the ids according to an estimation of risk;
 - setting adapters that return consistent ids.
 
-The ids of Morpho v1 lending markets could be for example the tuple `(CollateralToken, LLTV, Oracle)` and `CollateralToken` alone.
-A vault could be setup to enforce the following caps:
+The ids of Morpho v1 lending markets could be for example the market parameters `(CollateralToken, LoanToken, IRM, LLTV, Oracle)` and `CollateralToken` alone.
```

Comment:
```suggestion
The ids of Morpho v1 lending markets could be for example the market parameters `(LoanToken, CollateralToken, IRM, LLTV, Oracle)` and `CollateralToken` alone.
```

Reply:
This was to align with the Collat/Loan notation. Note that the other fields are not in the MarketParams order also. Do you think we should just stick to the MarketParams order ?